python/activator/activator.py

```diff
@@ -1,3 +1,25 @@
+# This file is part of prompt_prototype.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+__all__ = ["check_for_snap", "next_visit_handler"]
```

Comment:
Please include blank lines between the license blocks and the `__all__` definition; it's easy to miss the `__all__` with the current formatting.

Reply:
Should `activator.py` export any components at all? It was pretty clearly written as a Python script, not a library (see the `if __name__ == "__main__":` block at the bottom).

Reply:
I'm not sure I follow. You can import anything in Python, whether or not it's part of the module's API, and testing is a case where it's reasonable to import internals.

Reply:
There are so many module-level items in `activator.py`, I wanted to hide them all.